.github/workflows/update_groups_weekly_testnets.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           google-application-credentials: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           sh-attesters: hydra-s1-accountbound
           sh-attesters-networks: goerli mumbai
+          # sh-registry-tree-networks: goerli mumbai
```

Comment:
same?

Reply:
I just added a comment to specify that the variable value should be set, but it wasn't originally